db/python/layers/web.py

```diff
@@ -16,9 +17,29 @@
 from db.python.tables.analysis import AnalysisTable
 from db.python.tables.project import ProjectPermissionsTable
 from db.python.tables.sequence import SampleSequencingTable
+from models.base import SMBase
+
 from models.enums import SampleType, SequenceType, SequenceStatus, SequenceTechnology
 
 
+class MetaSearchEntityPrefix(Enum):
+    """Links prefixes to tables"""
+
+    p = 'participant'
+    s = 'sample'
+    sq = 'sequence'
+    f = 'family'
+
+
+class SearchItem(SMBase):
+    """Summary Grid Filter Model"""
+
+    model_type: str
```

Comment:
Is it this?

```suggestion
    model_type: MetaSearchEntityPrefix
```

Reply:
I think I'm using enums not quite right - If I do it as you suggest, the frontend complains that the model_type is a string instead of of type MetaSearchEntityPrefix when I try to pass it to the back. Would appreciate a very quick double check of this